src/MainWindow.vala

```diff
@@ -170,7 +173,18 @@ public class Sideload.MainWindow : Hdy.ApplicationWindow {
                 notification.set_body (_("The app was installed"));
             }
 
+            var icon = get_application_icon ();
+            if (icon != null) {
+                notification.set_icon (new ThemedIcon (icon));
+            }
             application.send_notification ("installed", notification);
         }
     }
+
+    private string? get_application_icon () {
```

Comment:
Instead of returning a string here, we should return a `GLib.Icon` since `set_icon ()` takes a `GLib.Icon` anyways